pyproject.toml

```diff
@@ -36,18 +36,32 @@ content-type = "text/markdown"
 
 [project.optional-dependencies]
 all = [
-    "vedo>=2023.4.3,<2023.5.0",
+    "vedo==2023.5.0",
```

Comment:
update? as >=?

Reply:
Needs more testing the current dev version gives me some errors while using the standard plotting funtions.